src/Microsoft.AspNetCore.Mvc.Core/AuthorizeFilter.cs

```diff
@@ -10,7 +10,7 @@
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Internal;
 
-namespace Microsoft.AspNetCore.Mvc.Internal
+namespace Microsoft.AspNetCore.Mvc
```

Comment:
The two other things that would move here right away are `IAllowAnonymous` and the filter equivalent of the same, so that gives us 3 things